assets/src/higher-order-components/filter-bar/entity-list-filter-bar.js

```diff
@@ -93,50 +102,88 @@ class EntityListFilterBar extends Component {
 	 * @return {Object} rendered grid view icon button
 	 */
 	gridView = ( view, setGridView ) => (
+		<Fragment>
+			<label
+				className="screen-reader-text"
+				htmlFor={ `ee-grid-view-btn-${ this.props.instanceId }` }>
+				{ __( 'list view', 'event_espresso' ) }
+			</label>
+			<IconButton
+				id={ `ee-grid-view-btn-${ this.props.instanceId }` }
+				className={ view === 'grid' ? 'active-list' : '' }
+				icon="screenoptions"
+				tooltip={ __( 'grid view', 'event_espresso' ) }
+				onClick={ setGridView }
+			/>
+		</Fragment>
+	);
```

Comment:
I'm wary of having a separate area for setting defaults.  I think we should _consider_:

- a unified options/settings toggle that would be used for the entire editor eventually for controlling various defaults  in the editor (default per page for things) etc.  What would that look like, how would options be registered with it.
- Does per-page and view type even NEED controllable defaults.  Could it not just remember what the user has selected in the existing toggles between sessions and across events? For example, user selects 5 as the per-page value for date list and 8 as the per-page value for ticket list.  They go and edit another event and that value is automatically used for that new event.  OR  could remembered specific to an event/user.